docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst

```diff
@@ -268,7 +265,7 @@ Using two versions of the CodeQL CLI
 
 If you want to use the latest CodeQL features to execute queries or CodeQL tests,
 but also want to prepare databases that are compatible with a specific version of
-LGTM Enterprise, you may need to install two versions of the CLI. The
+CodeQL code scanning on GitHub Enterprise Server, you may need to install two versions of the CLI. The
 recommended directory setup depends on which versions you want to install:
 
 - If both versions are 2.0.2 (or newer), you can unpack both CLI archives in the
```

Comment:
I'm guessing that we can simplify this end section by removing the reference to [CodeQL CLI 2.0.2](https://github.com/github/codeql-cli-binaries/releases/tag/v2.0.2)

Reply:
Hmmm, we might actually be able to ditch this altogether as we no longer refer to it from elsewhere. Let's consider this separately.

Reply:
👍🏻 It's probably worth looking through the CodeQL docs for mentions of specific versions and removing references to the oldest, unsupported versions.